index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Aimal Amiri</title>
+
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <nav class="container">
+        <a href="#" class="navbar-logo font-inter text-bold decoration-none">Aimal Amiri</a>
+        <ul class="navbar-menu">
+            <li class="navbar-link font-inter"><a href="#" class="text-white decoration-none">Portfolio</a></li>
+            <li class="navbar-link font-inter"><a href="#" class="text-white decoration-none">About</a></li>
+            <li class="navbar-link font-inter"><a href="#" class="text-white decoration-none">Contact</a></li>
+            <li class="navbar-link font-inter">
+                <a href="#" class="text-white decoration-none">
+                    <img src="./assets/svg/mail.svg" alt="mail" class="text-white" />
+                </a>
+            </li>
+        </ul>
+    </nav>
+    <header class="header">
+        <div class="social-media-links">
+            <a href="#" class="btn btn-circle"><img src="assets/svg/github.svg" alt="Github icon"></a>
+            <a href="#" class="btn btn-circle"><img src="assets/svg/linkedin.svg" alt="Linkedin icon"></a>
+            <a href="#" class="btn btn-circle"><img src="assets/svg/angel-list.svg" alt="Angel List icon"></a>
+            <a href="#" class="btn btn-circle"><img src="assets/svg/twitter.svg" alt="Twitter icon"></a>
+            <a href="#" class="btn btn-circle"><img src="assets/svg/medium.svg" alt="Medium Icon"></a>
+        </div>
```

Comment:
Please ensure that the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has `a link (a tag)` according to the requirement.